pysces/body.py

```diff
@@ -48,11 +49,78 @@ def cylinder(radius, num_points):
     return Body(points)
 
 def flat_plate(num_points):
-    """Return a flat plate with the given number of points"""
-    x = np.linspace(1, 0, num_points)
+    """
+    Return a flat plate with the given number of points.  In body coordinates
+    the plate runs from (0,0) to (1,0)."""
+    x = np.linspace(1, 0, num_points) # from 1 to 0 so trailing edge at index 0
     y = np.zeros_like(x)
     return Body(np.array([x, y]).T)
 
+def joukowski_foil(xcenter=-.1, ycenter=.1, a=1, numpoints=32):
+    """
+    Return a Joukowski foil centered with the trailing edge at (2a,0), and 
```

Comment:
See docstring comment above.  Also, might be good to have a consistent convention for specifying the chord of airfoils (e.g., the same keyword argument either for the chord, or the half-chord).  I'd probably suggest something like `chord=1` instead of `a=1` for instance.